### PR TITLE
Update ediscovery-troubleshooting-common-issues.md

### PR DESCRIPTION
**Pull request by syagey**
https://o365exchange.visualstudio.com/Enterprise%20Cloud/_workitems/edit/2139277

We receive multiple escalations for exports stuck where we recommend canceling and re-running the export.  We are attempting to provide additional deflection at the admin center level by suggesting a re-run when they have an export stuck issue.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md [hunk 1]
```diff
@@ -142,6 +142,15 @@ When exporting search results from eDiscovery or Content Search in the Security
 
 6. If you still have issues, consider dividing searches that return a large set of results into smaller searches. For example, you can use date ranges in search queries to return a smaller set of results that can be downloaded faster.
 
+## Error/issue: Exporting not progressing or stuck
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md]:
```suggestion
## Error/issue: Exporting not progressing or is stuck
```

File: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md [hunk 2]
```diff
@@ -142,6 +142,15 @@ When exporting search results from eDiscovery or Content Search in the Security
...
+
+Exporting search results from eDiscovery or Content Search in the Security and Compliance Center appear stuck or not progressing.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md]:
```suggestion
Exporting search results from eDiscovery or Content Search in the Security and Compliance Center appears stuck or is not progressing.
```

File: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md [hunk 3]
```diff
@@ -142,6 +142,15 @@ When exporting search results from eDiscovery or Content Search in the Security
...
+
+### Resolution
+
+1. Re-run the search
+2. Export the content
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md]:
```suggestion
1. Re-run the search.
2. Export the content.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @syagey, kindly commit the suggested minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Comment by markjjo** [comment]
@syagey PR is revised and merged. Can you please sign the contributor agreement? The PR is saying that it need to be signed.

**Pull request merged**